blockchain/chain_sync/src/sync.rs

```diff
@@ -133,7 +133,7 @@ where
 
 impl<DB> ChainSyncer<DB>
 where
-    DB: BlockStore,
+    DB: BlockStore + Sync + Send + 'static,
 {
```

Comment:
Sorry to comment on changes unreleated to yours, but this duplicate declaration is unneeded. There's no reason this is necessary, if you could remove the duplicate declaration and combine that would be awesome!